go.mod

```diff
@@ -7,6 +7,7 @@ replace (
 	github.com/crewjam/saml => github.com/rancher/saml v0.0.0-20180713225824-ce1532152fde
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
+	github.com/rancher/types v0.0.0-20190917172018-55dc8a6c4865 => /Users/rajashree/go-work/src/github.com/rancher/types
```

Comment:
I am assuming you will fix this in the final draft?

Reply:
yes..

Reply:
the types pr isn't reviewed or merged yet, so yeah once it is merged I will be updating go.mod with the commit